### ISSUE TITLE
Accessibility Testing

### ISSUE THREAD

**Issue by kierblk** [saramccombs/authentic-grace#16]
Run the website through a web accessibility tool and share the results, including what you could change to make the site more web accessible to all.

**Comment by kierblk** [comment]
[Link to Full Report: https://www.boia.org/a11yreports/?domref=saramccombs.github.io](https://www.boia.org/a11yreports/?domref=saramccombs.github.io)

### Summary Report:

A sampling of saramccombs.github.io's webpages was scanned using the Bureau of Internet Accessibility's automated scanning tool and the summary results are available below. Although automated testing can only identify 20-30% of the actual issues, it provides a great initial indication of overall accessibility.

**12/28 Checkpoints Failed**

**Perceivable:**
Information and user interface components must be presentable to users in ways they can perceive.

The following 5 of 13 failed:
[1.1.1 Nontext Content](https://www.boia.org/WCAG2/CP/1.1.1)
[1.3.1 Info and Relationships](https://www.boia.org/WCAG2/CP/1.3.1)
[1.3.5 Identify Input Purpose](https://www.boia.org/WCAG2/CP/1.3.5)
[1.4.3 Contrast (Minimum)](https://www.boia.org/WCAG2/CP/1.4.3)
[1.4.11 Non-Text Contrast](https://www.boia.org/WCAG2/CP/1.4.11)

**Operable:**
User interface components and navigation must be operable.

The following 3 of 9 failed:
[2.1.1 Keyboard](https://www.boia.org/WCAG2/CP/2.1.1)
[2.4.1 Bypass Blocks](https://www.boia.org/WCAG2/CP/2.4.1)
[2.4.4 Link Purpose (In Context)](https://www.boia.org/WCAG2/CP/2.4.4)

**Understandable:**
Information and the operation of user interface must be understandable.

The following 2 of 4 failed:
[3.1.1 Language of Page](https://www.boia.org/WCAG2/CP/3.1.1)
[3.3.2 Labels or Instructions](https://www.boia.org/WCAG2/CP/3.3.2)

**Robust:**
Content must be robust enough that it can be interpreted reliably by a wide variety of user agents, including assistive technologies.

The following 2 of 2 failed:
[4.1.1 Parsing](https://www.boia.org/WCAG2/CP/4.1.1)
[4.1.2 Name, Role, Value](https://www.boia.org/WCAG2/CP/4.1.2)

With the noting results from the application testing, there appears to be a storing need for more clarity in view and textual accessibilites in order for all consumers to be able to access the webpage. As a team we can also work on the multiple appearances of the webpage and providing more detailing per picture.
